net.pdl2ork.PdL2Ork.yml

```diff
@@ -0,0 +1,197 @@
+id: net.pdl2ork.PdL2Ork
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+default-branch: stable
+command: pd-wrapper.sh
+finish-args:
+# Display
+  - --share=ipc
+  - --socket=x11
+  #- --socket=wayland
+  - --device=dri
+# Audio
+  - --socket=pulseaudio
+  # currently disabled per a flathub maintainer's request
+  #- --system-talk-name=org.freedesktop.RealtimeKit1
+  - --filesystem=xdg-run/pipewire-0
+# Filesystem access (pd doesn't yet work with portals)
+  - --filesystem=home
+# MIDI access (Flatpak doesn't yet support exposing only MIDI devices)
+  - --device=all
+# Linux Audio plugins
+  - --env=LADSPA_PATH=/app/extensions/Plugins/ladspa:/app/lib/ladspa
+add-extensions:
+  org.freedesktop.LinuxAudio.Plugins:
+    directory: extensions/Plugins
+    version: '23.08'
+    add-ld-path: lib
+    merge-dirs: ladspa
+    subdirectories: true
+    no-autodownload: true
+
+cleanup:
+  - /include/*
+  - /lib/pd-l2ork/Makefile.am
+  - /lib/pd-l2ork/README.txt
+  - /lib/pd-l2ork/bin/todo.txt
+  - /lib/pd-l2ork/bin/nw/credits.html
+
+modules:
+- name: pd-l2ork
```

Comment:
Seems like some libraries are missing?

![image](https://github.com/flathub/flathub/assets/62639087/fb8de42d-1f51-4ae6-b8db-5e6c7dbb486f)

Reply:
This is just a default build. This would work on a regular (e.g. deb deployment) if the user has NDI hardware, as in that case they would also have libraries installed system-wide. So, the error would be still there if they don't. I just build it for convenience. Here I could disable it but would rather wait and see if there is a request to bundle it (assuming that the library is FOSS-friendly).